src/components/Icon/Icon.tsx

```diff
@@ -151,7 +152,8 @@ export type IconName =
   | "information"
   | "analytics"
   | "support"
-  | "apps";
+  | "apps"
+  | "domain_block"
```

Comment:
```suggestion
  | "ad_block"
```

pubconの事情が漏れているような気がしており...

Reply:
命名に関する話って @noronaoki さんとするんですかね？:thinking:

Reply:
これに関してはドメインブロックよりもアドブロックが合いそうです！

Reply:
@noronaoki 何か適切な命名あったりしますか？（カンプに追加する際にも名前は付けている認識だけど違うのか:thinking:

Reply:
ああ、すでに返答していましたね:pray:

Reply:
回答ありがとうございます🙏